mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -56,8 +71,9 @@ abstract class PcCollector[T](driver: InteractiveDriver, params: OffsetParams):
     case TypeApply(sel: Select, _) :: tail if sel.span.contains(pos.span) =>
       Interactive.pathTo(sel, pos.span) ::: rawPath
     case _ => rawPath
-
-  def collect(parent: Option[Tree])(tree: Tree, pos: SourcePosition): T
+  def collect(
+      parent: Option[Tree]
+  )(tree: Tree, pos: SourcePosition, symbol: Option[Symbol]): T
```

Comment:
We have additional parameter `symbol`. Sometimes tree doesn’t have correct `symbol`, so we first try to create `NodeInfo` using this. In Scala 2 we were creating `Ident(symbol)` and passing it instead of real tree, but I think this approach is better. It's used in import selectors and named arguments.